src/kernel/chainparams.cpp

```diff
@@ -76,17 +86,17 @@ class CMainParams : public CChainParams {
         consensus.signet_challenge.clear();
         consensus.nSubsidyHalvingInterval = 210000;
         consensus.script_flag_exceptions.emplace( // BIP16 exception
-            uint256S("0x00000000000002dc756eebf4f49723ed8d30cc28a5f108eb94b1ba88ac4f9c22"), SCRIPT_VERIFY_NONE);
+            uint256{"00000000000002dc756eebf4f49723ed8d30cc28a5f108eb94b1ba88ac4f9c22"}, SCRIPT_VERIFY_NONE);
```

Comment:
TODO: Measure.

Reply:
I also don't think this will be significant, will rent a cheap hetzner server and run something like:
```
hyperfine \
  --export-markdown make_bench.md \
  --warmup 1 --runs 10 \
  --parameter-list COMMIT 30cef5,be750d \
  --prepare 'git checkout {COMMIT} && git clean -fxd && git reset --hard && ./autogen.sh && ./configure --disable-ccache' \
  'echo {COMMIT} && make -j1'
  ```
  
i.e. 10 single-threaded, uncached compilations for 30cef5 vs be750d - would that be representative?

Reply:
> uncached compilations for 30cef5 vs be750d - would that be representative?

Maybe, but I don't think this is that useful. Anyone doing development, should be using `ccache`, only compiling the code they need, etc, and in that case, I don't think compilation is "way too slow". Anyone who isn't doing development, is likely only compiling once, where +/- N seconds/minutes in compilation time, doesn't really matter.

Reply:
```
Summary
  'echo 30cef5 && make -j1 && ./src/bitcoind --version | head -n 1' ran
    1.00 ± 0.00 times faster than 'echo be750d && make -j1 && ./src/bitcoind --version | head -n 1'
```
both of them take 45 minutes on a i7-7700, without cache.

Reply:
Thanks @paplorinc for helping put that worry to rest!